reference/JsSetHostPromiseRejectionTracker.md

```diff
@@ -0,0 +1,22 @@
+Sets a callback function that will be called to notify of unhandled Promise Rejections.
+### Syntax 
+```
+CHAKRA_API (JsErrorCode)
+    JsSetHostPromiseRejectionTracker(
+         _In_ JsHostPromiseRejectionTrackerCallback promiseRejectionTrackerCallback, 
+        _In_opt_ void *callbackState);
+```
+### Parameters 
+* __promiseRejectionTrackerCallback__: The callback function being set.
+* __callbackState__:  User provided state that will be passed back to the callback.
+
+### Return Value 
+The code **JsNoError** if the operation succeeded, a failure code otherwise.
+### Remarks 
```

Comment:
nit: Add a blank line before each section header to make the source more consistent and readable